fusionSuite/frontend
# misc: Better errors handling, handle types ids dynamically, update Angular

Another PR to fix or improve various things.

Changes proposed in this pull request:

- Add a handler to catch errors globally during the initialization
- Handle types ids dynamically by loading info from the `/v1/config/types` API
- Improve the loading of initial configuration (i.e. backendUrl)
- Update Angular to 14.2

How to test the feature manually:

1. Run the frontend without the backend, verify an error is displayed
2. Run the frontend with the backend, verify you can list and create organizations

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
